Dockerfile

```diff
@@ -28,13 +27,6 @@ RUN adduser \
     --uid "${UID}" \
     "${USER}"
 
-# Speed up build by leveraging docker layer caching
-COPY go.mod go.sum ./
-RUN go mod download
-
-ADD . /work
-RUN make
```

Comment:
We now build locally, which should be fine since we're building a static binary.